gearmand/tox.ini

```diff
@@ -2,7 +2,7 @@
 minversion = 2.0
 basepython = py38
 envlist =
-  py{27,38}
+  py{27,38}-{1.0.6,1.1.17,1.1.18}
```

Comment:
Why the specific choice of versions?

Reply:
`1.0.6` is the original version included in the check, so seemed appropriate to keep that one

`1.1.17` and `1.1.18` are the two most recent versions, so more likely to be a version that a current customer would be using.

Didn't include a `latest` because this setup requires an explicit version to be identified, and since we had pinned on an old version for so long, keeping at the bleeding edge didn't seem necessary.

Reply:
Ok, does it make sense to test both 1.1.17 and 1.1.18? Is there something in the changelog that suggests that we need to test both? We cannot test every minor release of every service we integrate with so unless there is a specific reason I'd stick with just 1.1.18 and I'm not sure if that's even necessary if the API we're using has not changed

Reply:
Doesn't look like much in the changelogs between the latest two, so I'll change it to just include `1.0.6` and `1.1.18`.